ShellyPy/gen1.py

```diff
@@ -27,9 +27,11 @@ def __init__(self, ip, port = "80", *args, **kwargs):
         super().__init__(ip, port, *args, **kwargs)
         self.__generation__ = 1
 
-    def update(self):
+    def update(self, index = 0):
```

Comment:
this is the wrong approach, update should take no arguments for anything device specific.